app/src/main/java/ca/llamabagel/transpo/trips/ui/TripDetailsFragment.kt

```diff
@@ -22,6 +35,21 @@ class TripDetailsFragment : Fragment() {
         return inflater.inflate(R.layout.trip_details_fragment, container, false)
     }
 
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        viewModel.setStop(StopId(args.stopId), args.trip)
+        viewModel.tripData.observe(this, Observer(::displayTrip))
+    }
+
+    private fun displayTrip(trip: Trip) {
+        view?.run {
+            findViewById<TextView>(R.id.heading).text = trip.destination
+            findViewById<TextView>(R.id.has_bike).text = "Has Bike Rack? ${trip.hasBikeRack}"
```

Comment:
This is temporary right?

Reply:
Yes, very much so. Will be corrected once the UI itself is worked on.